MsCorePkg/MacAddressEmulationDxe/MacAddressEmulationDxe.h

```diff
@@ -0,0 +1,73 @@
+/** @file
+
+  DXE Driver header for handling MAC Address Emulation.
+
+  Copyright (C) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#pragma once
```

Comment:
uefi code generally uses other pattern for protection.  Keep consistent.  Maybe someday we will change all of it.

Reply:
Sure wish VFR compiler understood `#pragma once` usually that's the argument "for" using `#ifdef`.

Maybe we'll get around to fixing the VFR compiler someday.